build.gradle

```diff
@@ -230,6 +230,9 @@ project('shared:security') {
         // in newer Java SE versions (9 and newer).
         compile group: 'javax.xml.bind', name: 'jaxb-api', version: jaxbVersion
         compile group: 'org.glassfish.jaxb', name: 'jaxb-runtime', version: jaxbVersion
+        // We need to explicitly import these as Bookkeeper 4.14+ is not transitively including them anymore.
+        compile group: 'org.bouncycastle', name: 'bcpkix-jdk15on', version: bouncyCastleVersion
```

Comment:
What do we use this for?

Reply:
In the previous version of Bookkeeper, we had these dependencies included transitively (`gradlew allDeps`):
```
    |         +--- org.bouncycastle:bcpkix-jdk15on:1.60
|    |         |    \--- org.bouncycastle:bcprov-jdk15on:1.60
|    |         +--- org.bouncycastle:bcprov-ext-jdk15on:1.60
```
However, in Bookkeeper 4.14, these are not included anymore:
```
|    |         +--- org.bouncycastle:bc-fips:1.0.2
```
The problem is that these are needed in Pravega with security enabled, as otherwise we get errors like when running tests or Pravega:
```
2021-06-10T09:48:16.2082894Z             Caused by:
2021-06-10T09:48:16.2083462Z             java.lang.NoSuchFieldError: xmss_with_SHA256
2021-06-10T09:48:16.2084627Z                 at org.bouncycastle.pqc.asn1.PQCObjectIdentifiers.<clinit>(Unknown Source)
2021-06-10T09:48:16.2127087Z                 at org.bouncycastle.jce.provider.BouncyCastleProvider.loadPQCKeys(Unknown Source)
2021-06-10T09:48:16.2129240Z                 at org.bouncycastle.jce.provider.BouncyCastleProvider.setup(Unknown Source)
```
Still, I will check in both are strictly necessary or we just need one of them.